src/js/stores/ui.js

```diff
@@ -266,6 +266,10 @@ define(function (require, exports, module) {
                 windowWidth = window.document.body.clientWidth,
                 windowHeight = window.document.body.clientHeight;
 
+            if (!centerOffsets) {
```

Comment:
It shouldn't solve the cloaking weirdness... but it should solve the errors `left of undefined` we've been seeing onReset and sometimes on startup.